wpeverest/user-registration
# UR-270 Enhance - Native Date Field

### All Submissions:






### Changes proposed in this Pull Request:

Previously, there was no localization for date field which show calender in native language. Introducing native date field in this PR.

### How to test the changes in this Pull Request:

1. Goto form builder and drag date field, In form options you can see date localize option and then select language for the date field.
2. Goto Frontend and check whether selected language is show in date field or not also verify whether it's saving in same language to check login the user and goto edit profile where you can see date field with native language. 

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Native Date Field.
